src/main/scala/io/iohk/ethereum/keystore/Wallet.scala

```diff
@@ -2,8 +2,13 @@ package io.iohk.ethereum.keystore
 
 
 import akka.util.ByteString
+import io.iohk.ethereum.crypto._
 import io.iohk.ethereum.domain.{Address, SignedTransaction, Transaction}
+import org.spongycastle.crypto.AsymmetricCipherKeyPair
 
-case class Wallet(address: Address, privKey: ByteString) {
-  def signTx(tx: Transaction): SignedTransaction = ???
+case class Wallet(address: Address, prvKey: ByteString) {
+  lazy val keyPair: AsymmetricCipherKeyPair = keyPairFromPrvKey(prvKey.toArray)
+
+  def signTx(tx: Transaction): SignedTransaction =
+    SignedTransaction.sign(tx, keyPair, None)
```

Comment:
I believe @adamsmo has discovered that Parity is not using the new signing scheme for ETC (no chain ID). Adam, can you confirm?